packages/hackathon/src/models/SheetData.spec

```diff
@@ -82,19 +82,6 @@ describe('SheetData', () => {
         )
       })
     })
-    test('registers a user', async () => {
-      const hackathon = data.currentHackathon
-      expect(hackathon).toBeDefined()
-      if (hackathon) {
-        const mockUser = mockAHacker('1')
-        const actual = await data.registerUser(hackathon, mockUser)
-        expect(actual._user_id).toEqual(mockUser.id)
-        expect(actual.hackathon_id).toEqual(hackathon._id)
-        expect(actual._updated).not.toEqual(noDate)
-        expect(actual.date_registered).not.toEqual(noDate)
-        expect(actual.attended).toEqual(true)
-      }
-    })
```

Comment:
Why did you have to remove this? Do we have a different method for registering users in the UI now?

Reply:
Its moved to sheets_client. Separation of concerns. The user registering flow was oddly split between sheets_client and sheetdata. Now it's consolidated. Noted this in the PR description.

Reply:
thanks for the explanation